kolibri/deployment/default/settings/base.py

```diff
@@ -139,7 +139,7 @@
     os.makedirs(CONTENT_STORAGE_DIR)
 
 # Base default URL for downloading content from an online server
-CENTRAL_CONTENT_DOWNLOAD_BASE_URL = "https://studio.learningequality.org"
+CENTRAL_CONTENT_DOWNLOAD_BASE_URL = "http://studio.learningequality.org"
```

Comment:
I think it's fine to download contents over the open internet -- but how do we ensure that there are no secrets shared via this URL?

:+1: for the naming of the URL, it does imply that it's only intended for content download... but might suffix it with `_INSECURE`, to be.... secure :tada: